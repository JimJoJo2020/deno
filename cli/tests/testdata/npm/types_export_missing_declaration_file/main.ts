// should error here
import "npm:@denotest/no-types-in-conditional-exports/sub";
