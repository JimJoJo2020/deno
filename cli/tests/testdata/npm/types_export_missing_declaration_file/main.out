Download http://localhost:4545/npm/registry/@denotest/no-types-in-conditional-exports
Download http://localhost:4545/npm/registry/@denotest/no-types-in-conditional-exports/1.0.0.tgz
Check file:///[WILDCARD]/cli/tests/testdata/npm/types_export_missing_declaration_file/main.ts
error: Uncaught Error: [TS7016] Could not find a declaration file for subpath ./sub in /[WILDCARD]/npm/registry/@denotest/no-types-in-conditional-exports/1.0.0/package.json. /[WILDCARD]/npm/registry/@denotest/no-types-in-conditional-exports/1.0.0/lib/foo.js implicitly has an 'any' type.
[WILDCARD]
